transcription/transcription_test.go

```diff
@@ -0,0 +1,55 @@
+package transcription
+
+import (
+	"errors"
+	"net/smtp" // mock
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
```

Comment:
The whole point of mocking is that the email *cannot* get rejected! We should talk about mocking at some point. Maybe I can go over it on Saturday.